Move overlays files to config directory

Alex-developer opened the pull request:
This PR moved the overlay data directories to the config directory.

*NOTE* this is a breaking change and will require a full reinstall 

A new alias has been added to the web server conf
The config_repo Makefile creates the required directories and copies the moon font into the font folder
The install script modified the permissions on the data files in config

EricClaeys commented:
@Alex-developer    It's up to you, but I think creating a new variable in variables.sh called something like `ALLSKY_OVERLAY=${ALLSKY_CONFIG}/overlay` (or $ALLSKY_OVERLAY_CONFIG ??), then using it everywhere would simplify the code some and make it easier to maintain if the directory every moves or is renamed.
Other than that, things look good.

Alex-developer commented:
> @Alex-developer It's up to you, but I think creating a new variable in variables.sh called something like `ALLSKY_OVERLAY=${ALLSKY_CONFIG}/overlay` (or $ALLSKY_OVERLAY_CONFIG ??), then using it everywhere would simplify the code some and make it easier to maintain if the directory every moves or is renamed. Other than that, things look good.

Eric

That make a lot of sense so I'll do that.

Also please DO NOT merge this PR as I forgot a large bit of it, I have not moved the config files ! I have only moved the data files. I'll sort this and add to the PR

Alex-developer commented:
@EricClaeys Ok I think I am done with this so can you please check it over and merge if you are happy.

- I have added two variables one for the overlay and one for the modules system as they are separate and it made no sense to have the module config in the overlay folder.
- I have modified the Makefile to sort the directories and files
- I have tested fresh installs on Buster and Bullseye lite
